packages/trader/src/_common/utility.ts

```diff
@@ -0,0 +1,19 @@
+let static_hash: string;
+export const getStaticHash = () => {
```

Comment:
Seems like this is not used anywhere in trader package

Reply:
Will check and remove